packages/block-library/src/post-excerpt/edit.js

```diff
@@ -83,28 +83,26 @@ export default function PostExcerptEditor( {
 			withoutInteractiveFormatting={ true }
 		/>
 	);
+	const excerptClassName = classnames( 'wp-block-post-excerpt__excerpt', {
+		'is-inline': ! showMoreOnNewLine,
+	} );
+	const excerptValue =
+		rawExcerpt ||
+		strippedRenderedExcerpt ||
+		( isSelected ? '' : __( 'No post excerpt found' ) );
 	const excerptContent = isEditable ? (
 		<RichText
-			className={
-				! showMoreOnNewLine &&
-				'wp-block-post-excerpt__excerpt is-inline'
-			}
+			className={ excerptClassName }
 			aria-label={ __( 'Post excerpt text' ) }
-			value={
-				rawExcerpt ||
-				strippedRenderedExcerpt ||
-				( isSelected ? '' : __( 'No post excerpt found' ) )
-			}
+			value={ excerptValue }
 			onChange={ setExcerpt }
 			tagName="p"
 		/>
 	) : (
-		( renderedExcerpt && (
-			<Disabled>
-				<RawHTML key="html">{ renderedExcerpt }</RawHTML>
-			</Disabled>
-		) ) ||
-		__( 'No post excerpt found' )
+		<p
+			className={ excerptClassName }
+			dangerouslySetInnerHTML={ { __html: excerptValue } }
+		/>
```

Comment:
Thanks for the follow-up, @ndiego.

This looks good, but it changes the previous behavior of the non-editable excerpt:

* We should use `renderedExcerpt` as a value here. The current value is for RichText.
* Don't hide "No post excerpt found" when selecting the block.

I think something like this should do the trick:

```jsx
<p
	className={ excerptClassName }
	dangerouslySetInnerHTML={
		!! renderedExcerpt ? { __html: renderedExcerpt } : undefined
	}
>
	{ ! renderedExcerpt ? __( 'No post excerpt found' ) : null }
</p>
```

@ntsekouras, do you remember the reason for using disabled for non-editable exceprts? As far as I know, the core doesn't allow links in excerpts.

Reply:
We definitely need to show the `renderedExcerpt` if is not editable. Also we need the `Disabled` as it's possible to contain links with `excerpt_more` filter.

Reply:
Thanks for reviewing, however I am a bit confused by the need for `renderedExcerpt`. When used, it adds an extra `<p>` tag, which would be incorrect markup. Here's a screenshot.

![image](https://user-images.githubusercontent.com/4832319/155581693-e7ca240e-ef56-45de-b13f-944c5097d0fa.png)

Perhaps we should use `strippedRenderedExcerpt`? This seems to work correctly. 🤔

Reply:
I think we can go with `strippedRenderedExcerpt` so we will not even need `Disabled` or `dangerouslySetInnerHTML`. I tried to think of any implication, but couldn't find any, so.. The main thing here was about `raw` vs `rendered`, we must not show `raw`.

Reply:
In that case, something like this should do the trick :) We still need to display the "No post excerpt found" text when the block is selected.

```jsx
<p
	className={ excerptClassName }
>
	{ strippedRenderedExcerpt || __( 'No post excerpt found' ) }
</p>
```

Reply:
Excellent, I will get this updated in a few minutes 🙌

Reply:
I have made the update and also rebased to hopefully fix the failing check.